NOTICE-binary

```diff
@@ -1,14 +1,13 @@
-Apache Pinot (incubating)
-Copyright 2018 and onwards The Apache Software Foundation
-
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
 
 // ------------------------------------------------------------------
 // NOTICE file corresponding to the section 4d of The Apache License,
-// Version 2.0, in this case for
+// Version 2.0, in this case for 
 // ------------------------------------------------------------------
 
+Apache Pinot (incubating)
```

Comment:
Probably put these lines to the top?

Reply:
This is done by following the instructions at https://cwiki.apache.org/confluence/display/PINOT/Creating+an+Apache+Release

Reply:
Yes, I know that. But that doesn't mean I have to follow what the script is generated. Check out other apache projects like spark:
https://github.com/apache/spark/blob/master/NOTICE-binary

Reply:
Sure. Updated. PTAL